MAUI/DataForm/floating-label-layout.md

```diff
@@ -0,0 +1,777 @@
+                        DataObject="{Binding ContactInfo}"
+                        AutoGenerateItems="False"
+                        GenerateDataFormItem="OnGenerateDataFormItem">
+    <dataForm:SfDataForm.Items>
+        <dataForm:DataFormTextItem FieldName="Name">
+            <dataForm:DataFormTextItem.ErrorLabelTextStyle>
+                <dataForm:DataFormTextStyle TextColor="RosyBrown" FontAttributes="Italic"/>
+            </dataForm:DataFormTextItem.ErrorLabelTextStyle>
+        </dataForm:DataFormTextItem>
+    </dataForm:SfDataForm.Items>
+</dataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+{
+    if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name")
+    {
+        e.DataFormItem.LayoutType = DataFormLayoutType.TextInputLayout;
+        e.DataFormItem.ErrorLabelTextStyle = new DataFormTextStyle { FontAttributes = FontAttributes.Bold, TextColor = Colors.IndianRed };
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Valid message label style
+You can customize the text of valid message label by setting the TextColor, FontFamily, FontSize, and FontAttributes properties of `ValidMessageLabelTextStyle` in DataFormItem.
+
+{% tabs %}
+{% highlight XAML %}
+<ContentPage 
+...
+xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+<dataForm:SfDataForm x:Name="dataForm" 
+                        LayoutType="TextInputLayout"
+                        DataObject="{Binding ContactInfo}"
+                        AutoGenerateItems="False"
+                        GenerateDataFormItem="OnGenerateDataFormItem">
+    <dataForm:SfDataForm.Items>
+        <dataForm:DataFormTextItem FieldName="Name">
+            <dataForm:DataFormTextItem.ValidMessageLabelTextStyle>
+                <dataForm:DataFormTextStyle TextColor="ForestGreen" FontAttributes="Italic"/>
+            </dataForm:DataFormTextItem.ValidMessageLabelTextStyle>
+        </dataForm:DataFormTextItem>
+    </dataForm:SfDataForm.Items>
+</dataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+{
+    if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name")
+    {
+        e.DataFormItem.LayoutType = DataFormLayoutType.TextInputLayout;
+        e.DataFormItem.ValidMessageLabelTextStyle = new DataFormTextStyle { FontAttributes = FontAttributes.Bold, TextColor = Colors.GreenYellow };
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Appearance customization
+
+#### Changing outline corner radius
+The corner radius of the container will be changed by setting `OutlineCornerRadius` property of `TextInputLayoutSettings` to double value.
```

Comment:
The corner radius of the container will be changed by setting the `OutlineCornerRadius` property of the `TextInputLayoutSettings` to double value.

Reply:
Content changed